druid-shell/src/platform/gtk/window.rs

```diff
@@ -1165,10 +1212,10 @@ impl WindowHandle {
                 .downcast::<gtk::Box>()
                 .unwrap();
 
-            let first_child = &vbox.get_children()[0];
+            let first_child = vbox.get_children().first().unwrap().clone();
```

Comment:
I don't understand this change. can you please explain more

Reply:
uhh, oh it was unrelated. I can remove it, it just seemed more explicit about the panic.

This is part of a path to move some things from my GTK4 fork into GTK3. In GTK4 you done have a general `get_children` only `first/last_child`. So when copy-pasting to the GTK3 branch this became `vbox.get_children().first()` etc. 

Its just an artifact from copy pasting, nothing meaningful. I can undo all of these if you want.

Reply:
Given this pr is related *Undefined behavior*, I would keep this separate